dev/defs/shapes/polygon.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Polygon",
+    "key": {
+      "int": 51,
+      "string": "polygon"
+    },
+    "extends": "shapes/parametric_path.json",
+    "properties": {
+      "points": {
+        "type": "uint",
+        "initialValue": "5",
+        "key": {
+          "int": 125,
+          "string": "points"
+        },
+        "description": "The number of points for the polygon."
+      },
+      "cornerRadius": {
+        "type": "double",
+        "initialValue": "0",
+        "key": {
+          "int": 126,
+          "string": "cornerradius"
+        },
+        "description": "The corner radius."
+      }
+    }
+  }
```

Comment:
I just copied these straight from the editor defs. Is that correct or is there an automated process for this?

Reply:
There's an update_defs.sh script in the dev folder that nabs them! But it only pulls from master, so it requires the new defs to be there (maybe we should add a branch argument to that update_defs.sh script).